src/ir/context.rs

```diff
@@ -1111,6 +1113,27 @@ impl<'ctx> BindgenContext<'ctx> {
         &self.options
     }
 
+    /// Output graphviz dot file.
+    pub fn emit_ir_graphviz(&self, path: String) -> io::Result<()> {
+        let file = try!(File::create(path));
+        let mut dot_file = Box::new(io::BufWriter::new(file)) as Box<io::Write>;
```

Comment:
Why does this need to be boxed? If it doesn't need to be, then we shouldn't box it.